ConsoleApplicationGoBoard/GoBoardMain.cpp

```diff
@@ -0,0 +1,17 @@
+#include "ConsoleFunctions.h"
+#include <iostream>
+
+using namespace ConsoleFuntions;
+
+void main()
+{
+	GoBoard Game(19);
+	Rules rules(Game);
+	while (1)
+	{
+		PrintBoard(Game);
+		playerMove(Game);
```

Comment:
see how weird that lower case 'p' looks here? it looks significant but its significance is unclear